ccd-definition/CaseField/CareSupervision/caseDocuments.json

```diff
@@ -128,7 +128,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "courtBundleList",
     "FieldType": "Collection",
-    "FieldTypeParameter": "CourtBundleV2",
+    "FieldTypeParameter": "HearingCourtBundle",
```

Comment:
so this will break existing cases, in a way that will be very annoying to fix, the best solution that I can advise is creating a new separate field for it. and then have everything reference this new field, the migration can then transform everything from the old field to the new field and then

Reply:
at the end of the migration after the transformation you can then remove the old field